projects/packages/jetpack-mu-wpcom/src/features/wpcom-comments/wpcom-comments.php

```diff
@@ -6,7 +6,7 @@
  * @package automattic/jetpack-mu-wpcom
  */
 
-if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
+if ( ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) && ( ! defined( 'IS_ATOMIC' ) || ! IS_ATOMIC ) ) {
```

Comment:
Nitpick: There's a method in Host [for it](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/status/src/class-host.php#L77). 

`!(new Host())->is_wpcom_platform()` is more readable. :)